### ISSUE TITLE
Margin reference update

### ISSUE THREAD

**Issue by eddelbuettel** [eddelbuettel/tint#46]
@cderv tweeted

![image](https://user-images.githubusercontent.com/673121/105780644-8b9b1000-5f36-11eb-8ada-fa7e5f2257c0.png)

so may need to update here too.

**Comment by cderv** [comment]
Yes I think you'll also need to fix https://github.com/rstudio/tufte/issues/91:

With Pandoc 2.11+
 
````markdown
---
title: "Margin references"
output:
  tint::tintHtml: default
bibliography: test.bib
link-citations: yes
---

```{r}
knitr::write_bib(c("knitr", "tint"), "test.bib")
```

See @R-tint and @R-knitr
````

![image](https://user-images.githubusercontent.com/6791940/105822071-e296f880-5fbb-11eb-91e3-b0c4dd6fc83d.png)
